src/adapters/tests/leasing-adapter/offers.test.ts

```diff
@@ -77,6 +77,51 @@ describe(leasingAdapter.getOffersForContact, () => {
     assert(result.ok)
     expect(result.data).toEqual([expect.objectContaining({ id: offer.id })])
   })
+
+  it('should return 404 on not found', async () => {
+    nock(config.tenantsLeasesService.url)
+      .get('/contacts/P174965/offers')
+      .reply(404, {
+        content: {
+          ok: false,
+          err: 'not-found',
+        },
+      })
```

Comment:
Gör egentligen inget i det här fallet, men leasingrouten kommer väl inte returnera `{ ok: false, err: 'not-found' }`?